### ISSUE TITLE
no Setting if dataType is 6

### ISSUE THREAD

**Issue by cn72** [hacki11/ioBroker.bsblan#26]
when the Parameter bsb.dataType is 6 , the setting does not work.
Log:
```
bsblan.0 | 2020-12-15 21:13:58.120 | error | (6584) Error writing value: Could not set value: "{"5":{"status":0}}"
bsblan.0 | 2020-12-15 21:13:58.087 | info | (6584) Sending write request for bsblan.0.Sommerzeitbeginn_Tag/Monat_(5) (value: 25.03)
```

**Comment by cn72** [comment]
adding pull request, adding default handling.

**Comment by hacki11** [comment]
My last information was 5 and 6 are not writable: https://github.com/fredlcore/bsb_lan/issues/76#issuecomment-606149288

Was there a change? Did you test it successfully?

**Comment by cn72** [comment]
Yes. It is the Parameter 5 (Summertime begin) and Parameter 6 (Summertime end).
I set it successfully.
```
...
"native": {
    "id": "5",
    "bsb": {
      "name": "Sommerzeitbeginn Tag/Monat",
      "dataType_name": "SUMMERPERIOD",
      "dataType_family": "DDMM",
      "possibleValues": [],
      "isswitch": 0,
      "dataType": 6,	<= this value
      "readonly": 0,
      "unit": ""
    }
...
```
I have to add the Parameter to config.js to make it writeable.
In my special case the Parameter is using INF. But this is an exception. Normaly it is using SET.
By this way take a look to #24 to make it writeable in ioBroker looking for key readonly (dependent of the settings in BSB-LAN firmware by the user(DEFAULT_FLAG)) and #25 to handle unknown Parameter in config.js and set the writeType to SET for default.

**Comment by hacki11** [comment]
Fixed in 0.2
